### PR TITLE
Added wild boar to forests

### PR DESCRIPTION
**Pull request by RarkGrames**
#### Summary
```SUMMARY: Content "Added wild boar to forests"```

#### Purpose of change
Making wildlife more varied by adding a new animal.

#### Describe the solution
Adding entries to mammal.json and monstergroups.json.
Sources used: [stats](https://en.wikipedia.org/wiki/Wild_boar#Physical_description), [presence in NH](https://www.wildlife.state.nh.us/wildlife/feral-hogs.html), [population data - page 5.](https://www.wildlife.state.nh.us/wildlife/documents/wj-feral-hogs.pdf)

#### Testing
Spawned wild boars and piglets, killed them to check the butchering harvest lists.

### CODE REVIEW

**Comment by KorGgenT** [comment]
how common are wild boars in new england?

**Comment by RarkGrames** [comment]
> 
> 
> how common are wild boars in new england?

"At this time, feral swine populations in the north-eastern United States appear to be relatively sparse. However,  there  are  confirmed  populations  in  New Hampshire, New Jersey, New York and Pennsylvania."

File: data/json/monsters/mammal.json [hunk 1]
```diff
@@ -119,6 +119,78 @@
...
+    "id": "mon_boar_wild",
+    "type": "MONSTER",
+    "name": "wild boar",
+    "description": "Originally not native in the US, this wild omnivore is a crossbreed between the Eurasian wild boar escaped from hunting reserves and the domesticated pig went feral in the wilderness.  Its population has skyrocketed in the US during the last few decades.",
+    "looks_like": "mon_pig",
+    "default_faction": "pig",
+    "bodytype": "pig",
+    "categories": [ "WILDLIFE" ],
+    "species": [ "MAMMAL" ],
+    "volume": "100000 ml",
```
Comment (Night-Pryanik) [inline data/json/monsters/mammal.json]: I guess this could be converted to liters for convenience.

Comment (RarkGrames) [reply data/json/monsters/mammal.json]: Ok, changed it to liters.

**Pull request merged**